hazelcast/src/main/java/com/hazelcast/spi/merge/SplitBrainEntryViews.java

```diff
@@ -18,6 +18,7 @@
 
 import com.hazelcast.cardinality.impl.hyperloglog.HyperLogLog;
 import com.hazelcast.collection.impl.collection.CollectionItem;
+import com.hazelcast.map.impl.record.Record;
```

Comment:
`.impl` package inside public spi?

Reply:
Hmm, is this a big deal? `SplitBrainEntryViews` is a helper class which will in the end have all kind of record or item classes as parameters to construct `SplitBrainMergeEntryView` with all available data.

Do I have to move this class to the `impl` package of the SPI?

This goes hand in hand with the question, if the internal `SimpleSplitBrainEntryView` class needs to be public or if `SplitBrainMergeEntryView` needs setter methods. At the moment you cannot create an `SplitBrainMergeEntryView` instance for your custom service without implementing the interface on your own.

Nevertheless we should do all these refactoring in the end, since we have a bunch of PRs pending, which depend on the class as it is.

Reply:
Maybe we can put it to 'internal' package

Reply:
That's what I meant, there is an `spi.impl` package which seems to be suitable. So we could move it to `com.hazelcast.spi.impl.merge.SplitBrainEntryViews` or so.